pgjdbc/src/main/java/org/postgresql/gss/GSSOutputStream.java

```diff
@@ -42,12 +42,12 @@ private static int getBufferSize(GSSContext gssContext, MessageProp messageProp,
   @Override
   protected void flushBuffer() throws IOException {
     if (count > 0) {
-      writeWrapped(buf, 0, count);
+      writeWrapped(0, count);
       count = 0;
     }
   }
 
-  private void writeWrapped(byte[] b, int off, int len) throws IOException {
```

Comment:
The actual bug was that the parameter was named `b` while the code below was using `.wrap(buf`.
The fix is to use `.wrap(b, ..)`